src/pydicom/pixels/utils.py

```diff
@@ -1293,17 +1356,27 @@ def pack_bits(arr: "np.ndarray", pad: bool = True) -> bytes:
         Added the `pad` keyword parameter and changed to allow `arr` to be
         2 or 3D.
 
+    .. versionchanged:: 3.1
+
+        Allow packing of bytes and bytearray objects when numpy is not
+        installed.
+
     Parameters
     ----------
-    arr : numpy.ndarray
-        The :class:`numpy.ndarray` containing 1-bit data as ints. `arr` must
-        only contain integer values of 0 and 1 and must have an 'uint'  or
-        'int' :class:`numpy.dtype`. For the sake of efficiency it's recommended
+    arr : numpy.ndarray | bytes | bytearray
+        The :class:`numpy.ndarray`, `bytes`, or `bytearray` containing 1-bit
+        data as ints/bytes. If it is a :class:`numpy.ndarray`, `arr` must only
+        contain integer values of 0 and 1 and must have an 'uint'  or 'int'
+        :class:`numpy.dtype`. If it is a `bytes` or `bytesarray`, each byte
```

Comment:
`bytearray`

Reply:
resolved in [c1e9a0f](https://github.com/pydicom/pydicom/pull/2216/commits/c1e9a0f2c1b8225b608765a11be870fdd5d7fc99)